manifester/manifester.py

```diff
@@ -12,17 +12,23 @@
 
 class Manifester:
     def __init__(self, manifest_category, allocation_name=None, **kwargs):
-        self.allocation_name = allocation_name or "".join(random.sample(string.ascii_letters, 10))
-        self.offline_token = kwargs.get("offline_token", settings.offline_token)
-        manifest_data = settings.manifest_category.get(manifest_category)
-        self.subscription_data = manifest_data.subscription_data
-        self.sat_version = kwargs.get("sat_version", manifest_data.sat_version)
+        self.allocation_name = allocation_name or "".join(
+            random.sample(string.ascii_letters, 10)
```

Comment:
Consider adding a fauxfactory dependency for this.

Reply:
right now it is just a single call, but if more random data is needed later, that is a possibility. or the more performant
```python
str(uuid4()).split("-")[-1]
```